### PR TITLE
[MIG] web_refresher: Migration to 16.0

### PR DESCRIPTION
**Pull request by hugosantosred**

### CODE REVIEW

File: web_refresher/__init__.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+##############################################################################
+#
+#    Copyright (C) 2019 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Samuel Fringeli <samuel.fringeli@me.com>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
```
Comment (legalsylvain) [inline web_refresher/__init__.py]: This empty file doesn't require that copyright text. you can let empty.

File: web_refresher/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Web Refresher",
+    "version": "16.0.1.0.0",
+    "author": "Compassion Switzerland, Odoo Community Association (OCA)",
```
Comment (legalsylvain) [inline web_refresher/__manifest__.py:4]: adopt an addon ?

**Review by legalsylvain — APPROVED** [review]
Hi. thanks for porting this module !
2 non blocking point inline.
Otherwise, LGTM. Runboat test. works as expected. Thanks.

**Comment by hugosantosred** [comment]
Thanks for the quick review @legalsylvain!

Changes done

File: web_refresher/__manifest__.py [hunk 3]
```diff
@@ -1,11 +1,11 @@
 {
     "name": "Web Refresher",
-    "version": "15.0.1.0.0",
-    "author": "Compassion Switzerland, Odoo Community Association (OCA)",
+    "version": "16.0.1.0.0",
+    "author": "Compassion Switzerland, FactorLibre, Odoo Community Association (OCA)",
```
Comment (pedrobaeza) [inline web_refresher/__manifest__.py]: This migration doesn't qualify for being co-authors of the module. You should add yourself to contributors instead.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Functional review OK, but check the comment inline.

File: web_refresher/__manifest__.py (hunk 3 above)
Comment (hugosantosred) [reply web_refresher/__manifest__.py]: ok! Thanks for the review!, changes are already done

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
This patch is missing from versions 15 and 16: #2255

Maybe some of you can include it.

**Pull request merged**